### ISSUE TITLE
Error on exit

### ISSUE THREAD

**Issue by Lopi-py** [j-hui/fidget.nvim#4]
Fidget throws an error while showing the progress and exit

```
Error executing vim.schedule lua callback: ...m-data\site\pack\packer\start\fidget.nvim/lua/fidget.lua:116: Vim:E322: line number
out of range: 1 past the end
stack traceback:
        [C]: in function 'nvim_buf_set_lines'
        ...m-data\site\pack\packer\start\fidget.nvim/lua/fidget.lua:116: in function 'show'
        ...m-data\site\pack\packer\start\fidget.nvim/lua/fidget.lua:41: in function 'render_fidgets'
        ...m-data\site\pack\packer\start\fidget.nvim/lua/fidget.lua:75: in function 'fmt'
        ...m-data\site\pack\packer\start\fidget.nvim/lua/fidget.lua:232: in function 'handler'
        C:\tools\neovim\Neovim\share\nvim\runtime/lua/vim/lsp.lua:735: in function 'cb'
        vim.lua:285: in function <vim.lua:285>
```

## Steps to reproduce
`nvim a.lua`
exit from neovim with `:quitall` while fidget is rendering the lsp progress

**Comment by j-hui** [comment]
The error is coming from [here](https://github.com/neovim/neovim/blob/master/src/nvim/memline.c#L3111) in Neovim.

I'm not sure if I'm forgetting to do something that would avoid this race condition, so I'll have to ask around and see.

**Comment by Lopi-py** [comment]
Closed due to #18
